it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/ControllersSupportWelshTranslationISpec.scala

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.minorentityidentificationfrontend.controllers
+
+import org.jsoup.nodes.Document
+import org.scalatest.prop.TableDrivenPropertyChecks.forAll
+import org.scalatest.prop.{TableFor1, TableFor2, Tables}
+import play.api.test.Helpers.{OK, await, defaultAwaitTimeout}
+import uk.gov.hmrc.minorentityidentificationfrontend.assets.TestConstants._
+import uk.gov.hmrc.minorentityidentificationfrontend.featureswitch.core.config.{EnableFullTrustJourney, FeatureSwitching}
+import uk.gov.hmrc.minorentityidentificationfrontend.stubs.{AuthStub, StorageStub}
+import uk.gov.hmrc.minorentityidentificationfrontend.utils.ComponentSpecHelper
+import uk.gov.hmrc.minorentityidentificationfrontend.utils.ViewSpecHelper.ElementExtensions
+
+class ControllersSupportWelshTranslationISpec
+  extends ComponentSpecHelper
+    with StorageStub
+    with AuthStub
+    with FeatureSwitching{
+
+  private def baseUrl = "/identify-your-trust"
```

Comment:
We are only testing the trust controllers here. We should really add tests for Overseas and unincorporated association controllers. Probably, easiest to add to the main table loop.

Reply:
done